lib/pipeline.js

```diff
@@ -16,7 +16,9 @@ const REGEX_CAPTURING_GROUP = {
     pr: 1, // PR-1
     job: 2 // main or undefined if using legacy
 };
-const MAX_COUNT = 1000;
+const MAX_METRIC_GET_COUNT = 1000;
+const MAX_JOB_DELETE_COUNT = 10;
```

Comment:
wondering why 10 for job delete, but it's okay we can change later

Reply:
This is because the maximum number of concurrent processes is `MAX_JOB_DELETE_COUNT` * `MAX_BUILD_DELETE_COUNT` when pipelines are removed. We expect one of the two value to be smaller than 100.